nixos/modules/services/backup/borgbackup.nix

```diff
@@ -279,6 +291,46 @@ in {
             example = "-u +%s";
           };
 
+          persistent = mkOption {
+            type = types.bool;
+            default = false;
+            description = ''
+              Whether backups should be triggered if they would have triggered
+              at least once while inactive, e.g., when the machine was powered
+              off.
+            '';
+          };
+
+          randomizedDelaySec = mkOption {
+            type = types.str;
+            default = "0";
+            description = ''
+              Backups are delayed by a uniformly random amount of time up to
+              the value. Useful to stagger remote backups on a weak network
+              connection. Must be in the format described in
+              <citerefentry><refentrytitle>systemd.time</refentrytitle>
+              <manvolnum>7</manvolnum></citerefentry>. Either a unit-less value
+              (in seconds) or a time span value, e.g.,
+              <literal>"5min 20s"</literal>.
+            '';
+            example = "5 min";
+          };
+
+          restartSec = mkOption {
+            type = with types; nullOr (either str (listOf str));
```

Comment:
What would a list of strings mean?

Reply:
I think I may have copied it blindly from `startAt`? Not sure, reading the systemd documentation it doesn't look like it would make sense. Good catch!